Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/HandConstraint.cs

```diff
@@ -416,8 +416,15 @@ private static Ray CalculateProjectedSafeZoneRay(Vector3 origin, IMixedRealityCo
                 default:
                 case SolverSafeZone.UlnarSide:
                     {
-                        direction = Vector3.Cross(CameraCache.Main.transform.forward, Vector3.up);
-                        direction = IsPalmFacingCamera(hand) ? direction : -direction;
+                        if (rotationBehavior == SolverRotationBehavior.LookAtTrackedObject)
```

Comment:
Could you explain why we are setting direction to transform.right if the behavior is LookAtTrackedObject? Is it a fix for another issue you found?

Reply:
Certainly!

So the the reason I changed this is because the follow tracked object rotation behavior sets the position at a constant offset relative to the camera to object vector. 

In my opinion, LookAtTrackedObject should be positioned relative the object's rotation, regardless of the camera's position. This is the meat of this PR to be honest.

However, I realize this may in fact change the behavior MRTK user have come to expect, and so perhaps we should make this change as part of a new rotation behavior enum.